rpcs3/Emu/Audio/XAudio2/XAudio28Thread.cpp

```diff
@@ -57,6 +65,8 @@ void XAudio2Thread::xa28_destroy()
 		s_tls_xaudio2_instance->StopEngine();
 		s_tls_xaudio2_instance->Release();
 	}
+
+	CoUninitialize();
```

Comment:
That stackoverflow answer is rubbish, if you coinit/uninit in each thread then you won't get a crash "because other threads are using it", that is exactly why you must coinit/uninit in each thread!